subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -375,10 +375,21 @@
                 </tr>
             </thead>
             <tr>
+                <td>implementation</td>
+                <td>api</td>
```

Comment:
This is not the case. This relationship is only added by the `java-library` plugin and should thus be explained there.